### PR TITLE
#126214 - update testing env variables and add tests for Tribe__Ticke…

### PR DESCRIPTION
**Pull request by jesseeproductions**
…ts__RSVP__Attendance_Totals

_Ref:_ [C#126214](https://central.tri.be/issues/126214)

### CODE REVIEW

File: tests/wpunit/Tribe/Tickets/Attendance_RSVP_TotalsTest.php [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+	/**
+	 * @test
+	 */
+	public function it_should_count_total_rsvps_correctly() {
+
+		$event_id          = $this->factory()->event->create();
+		$rsvp_id           = $this->create_rsvp_ticket( $event_id );
+		$created_attendees = $this->create_many_attendees_for_ticket( 10, $rsvp_id, $event_id );
+
+		$Tickets__Attendance = new Tickets__Attendance( $event_id );
```
Comment (bordoni) [inline tests/wpunit/Tribe/Tickets/Attendance_RSVP_TotalsTest.php]: We should create a `make_instance()` and  `it_should_be_instanceable` tests for this class. Also, avoid the upper case on the variable names.

**Comment by jesseeproductions** [comment]
Update per feedback, including another similar test.

File: tests/wpunit/Tribe/Tickets/AttendanceTotalsTest.php [hunk 2]
```diff
@@ -48,8 +66,8 @@ public function it_should_count_checked_in_attendees_correctly() {
 			tribe( 'tickets.rsvp' )->checkin( $attendee, true );
 		}
 
-		$Tickets__Attendance = new Tickets__Attendance( $event_id );
-		$checked_in          = $Tickets__Attendance->get_total_checked_in();
+		$tickets__attendance =  $this->make_instance( $event_id );
```
Comment (sc0ttkclark) [inline tests/wpunit/Tribe/Tickets/AttendanceTotalsTest.php]:
```suggestion
		$tickets__attendance = $this->make_instance( $event_id );
```

File: tests/wpunit/Tribe/Tickets/AttendanceTotalsTest.php [hunk 3]
```diff
@@ -64,8 +82,8 @@ public function it_should_count_not_checked_in_attendees_correctly() {
 		$rsvp_id           = $this->create_rsvp_ticket( $event_id );
 		$created_attendees = $this->create_many_attendees_for_ticket( 10, $rsvp_id, $event_id );
 
-		$Tickets__Attendance = new Tickets__Attendance( $event_id );
-		$not_checked_in      = $Tickets__Attendance->get_total_not_checked_in();
+		$tickets__attendance =  $this->make_instance( $event_id );
```
Comment (sc0ttkclark) [inline tests/wpunit/Tribe/Tickets/AttendanceTotalsTest.php]:
```suggestion
		$tickets__attendance = $this->make_instance( $event_id );
```

File: tests/wpunit/Tribe/Tickets/AttendanceTotalsTest.php [hunk 4]
```diff
@@ -88,8 +106,8 @@ public function it_should_count_deleted_attendees_correctly() {
 			tribe( 'tickets.rsvp' )->delete_ticket( $event_id, $created_attendees[ $x ] );
 		}
 
-		$Tickets__Attendance = new Tickets__Attendance( $event_id );
-		$total_deleted       = $Tickets__Attendance->get_total_deleted();
+		$tickets__attendance =  $this->make_instance( $event_id );
```
Comment (sc0ttkclark) [inline tests/wpunit/Tribe/Tickets/AttendanceTotalsTest.php]:
```suggestion
		$tickets__attendance = $this->make_instance( $event_id );
```

**Comment by jesseeproductions** [comment]
@bordoni please check again so I can merge, thanks!

**Pull request merged**